src/run/paths.ts

```diff
@@ -114,3 +113,32 @@ async function expandFeaturePaths(
   featurePaths = [...new Set(featurePaths)] // Deduplicate the feature files
   return await expandPaths(cwd, featurePaths, '.feature')
 }
+
+async function deriveSupportPaths(
+  cwd: string,
+  featurePaths: string[],
+  unexpandedRequirePaths: string[],
+  unexpandedImportPaths: string[]
+): Promise<{
+  requirePaths: string[]
+  importPaths: string[]
+}> {
+  if (
+    unexpandedRequirePaths.length === 0 &&
+    unexpandedImportPaths.length === 0
+  ) {
+    const defaultPaths = getFeatureDirectoryPaths(cwd, featurePaths)
+    const requirePaths = await expandPaths(cwd, defaultPaths, '.js')
+    const importPaths = await expandPaths(cwd, defaultPaths, '.mjs')
```

Comment:
What about also supporting `.cjs` extension?

Reply:
Oh good point, that's still backwards-compatible as a default. Done!